### ISSUE TITLE
PROPOSAL: UNICEF Partner Identifier XM-UNICEF

### ISSUE THREAD

**Issue by amy-silcock** [org-id/register#486]
Proposal received by UNICEF for inclusion into org-id

## Proposal

**What open data standard are you working with? (e.g. 'OCDS', '360Giving', 'BODS' etc.)**
IATI

**The name of the list (or the organization that manages the list)**
UNICEF Partner Identifiers
This is a list of partners verified and registered in UNICEF’s partner database (with unique IDs). The list is managed by UNICEF.

**A suggested code for the list (optional)**
XM-UNICEF

**A short description of the list**
UNICEF is mandated by the United Nations General Assembly to advocate for the protection of children's rights, to help meet their basic needs and to expand their opportunities to reach their full potential, from early childhood through adolescence. UNICEF works with partners from the public sector, private sector, United Nations and civil society in over 190 countries and territories. UNICEF manages its own partner database and all registered partners are verified and assigned a unique organisation identifier.

**The country or countries that the list covers**
Global – countries are included in the list for each registered partner. 
https://www.unicef.org/where-we-work

**The legal form or organizations that the list covers**
Verified partners registered with UNICEF and assigned with unique IDs. (attached)
https://www.unicef.org/partnerships

**Any specific sectors that the list covers**
Development and humanitarian (more details can be found from this link)
https://www.unicef.org/what-we-do

**A URL for information on the list, and a URL for looking up identifiers (if available)**
https://open.unicef.org/unicef-partner-database

**Any information on available open data copies of this list**
n/a

**One or more examples of identifier from this list, and how you found them**

Partner name: DEPARTMENT OF EDUCATION LAGHMAN
Proposed code: XM-UNICEF-41122-217167

Partner name: FUNDACION EDUCACION Y COOPERACION
Proposed code: XM-UNICEF-41122-241611

**Comment by kd-ods** [comment]
Thanks, @amy-silcock. This information looks good. 

However 'XM' is not the right code to use in this case. ([XM](https://org-idguide-handbook.readthedocs.io/en/latest/metadata/#jurisdiction-code) is for lists where entries are multilateral or international agencies.) As this is a list created by an IATI publisher, I suggest we use ZZ-UNICEF.

Can I leave it to the IATI Helpdesk to create a pull request?

**Comment by stevieflow** [comment]
I'd like to query the practicality and purpose of this

The aim of org-id is to create an accessible list of _common_ registries from which specific organisations can then be identified

Usually these are jurisdiction-based registries for companies, charities and other entities, where registration is an outcome of a legal process.  Other instances exist - eg XM-DAC - where the organisations in the registry are in common use by others (in that case, CRS and IATI data).  

This proposal seems to just be an output from an internal vendor/partner database.  A logical progression from this would be that _any_ organisation could then request their _own_ org-id listing, which they can then deploy in _their_ data.  This seems to be contrary the spirit of referencing organisations on commonly recognised registries.  In turn, if organisations persist in using such internal namespaces, the task of **using** the data to understand and cross-reference partners and networks would get even more challenging.

Whilst it is no doubt that the type and range of partners on this list represent a wide range of very specific local organisations and entities, it seems the adoption of an internally focused prefix misses many of the issues the wider community are dealing with.  I am concerned this sets a precedent that would be detrimental to productive data use.

**Comment by HermanvanLoon** [comment]
I fully agree with the viewpoint of @stevieflow above. Key to have meaningful information is that the organization can be worldwide uniquely identified regardless of which publisher is referring to an organization. Moving in the direction which is now proposed for UNICEF will mean that we will never be able to answer the question 'What is organization X doing and which donors are contributing' since there is no common IATI identifier for organization X anymore. This would be very detrimental to the usability of IATI data in general and a major setback since we introduced strict rules for defining organization identifiers (in version 2.01 of the standard if I am not mistaken).

**Comment by kd-ods** [comment]
@amy-silcock - you asked on another channel for me to clarify what org-id's process is when community members raise concerns about the addition of a list.

As you say, org-id.guide is a resource whose use extends beyond just the IATI community. We therefore don't seek to 'police' entries, beyond making sure that they have enough information to be usable. It is for communities of publishers and users of their data to determine which registries or lists are most useful and fitting for their needs. The feedback here suggests that the UNICEF list may not be as useful to those publishing or using IATI data as hoped.

We can add the UNICEF list to org-id.guide, but the concerns of your community members will remain.

For org-id's part: we could offer more guidance and a rationale for what makes a 'good quality' identifier in open data. This is currently not well covered in the [handbook](https://org-idguide-handbook.readthedocs.io/en/latest/about/) but would - I think - be a useful resource for open data communities.

**Comment by markbrough** [comment]
Hi all -- just wanted to flag that last week, we shared a methodology on IATI Connect for identifying government entities, which may be relevant for this discussion:
https://iaticonnect.org/group/7/discussion/new-methodology-identifying-government-entities

**Comment by Mishaseino** [comment]
@stevieflow @HermanvanLoon 
Many thanks for your constructive comments! We understand the perspective raised and potential implications. We have gone back and re-thought the strategy and decided that we will not go ahead with a request for creation of a UNICEF-specific unique ID.

**Comment by IsabelBirds** [comment]
Closing as superseded by #560
